includes/WikiManager.php

```diff
@@ -377,8 +377,18 @@ private function logEntry( string $log, string $action, string $actor, string $r
 	}
 
 	private function recacheJson( $wiki = null ) {
-		$cWJ = new CreateWikiJson( $wiki ?? $this->config->get( 'CreateWikiGlobalWiki' ), $this->hookRunner );
+		$cWJ = new CreateWikiJson($wiki ?? $this->config->get('CreateWikiGlobalWiki'), $this->hookRunner);
 		$cWJ->resetDatabaseList();
 		$cWJ->update();
 	}
+	public function getUserRequestCount(int $userID, int $viewLevel) {
+
+		return $this->cwdb->newSelectQueryBuilder()
+			->select('*')
```

Comment:
Related to the comment below, don't just query for everything since that's just a waste of bandwidth in this case, but you're going to need cw_visibility for what you're going to do later.
```suggestion
			->select( 'cw_visibility' )
```